eng/pipelines/mgmt-ci.yml

```diff
@@ -24,7 +24,7 @@ trigger:
       - sdk/core/core-xml/
       - sdk/core/logger/
       - sdk/cosmosdb/cosmos/
-      - sdk/digitaltwins/digital-twins/
+      - sdk/digitaltwins/digital-twins-core/
```

Comment:
We have both `digital-twins` and `digital-twins-core` packages. So, why are we replacing instead of adding?

Reply:
I am not sure why we have `digital-twins`, their directory is almost empty: https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/digitaltwins/digital-twins

Reply:
I will remove that directory in this PR.